src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesApiService.kt

```diff
@@ -27,4 +29,12 @@ class ActivitiesApiService(@Qualifier("activitiesApiWebClient") private val webC
       .retrieve()
       .bodyToMono(AllocationMigrateResponse::class.java)
       .awaitSingle()!!
+
+  suspend fun getActivityCategories(): List<String> =
+    webClient.get()
+      .uri("/activity-categories")
+      .retrieve()
+      .bodyToMono<List<ActivityCategory>>()
+      .awaitSingle()
```

Comment:
tiny style thing, I did a bit for transformation like this before e.g `.map {it.code}` but decided to move up to the caller.

Basically I had an issue when I tried to refactor and collapse the bodyToMono and awaitSingle to awaitBody.


Obviously you will be fine but thought it was wroth mentioning